### ISSUE TITLE
Update to angular@6.0 and rxjs@6.1

### ISSUE THREAD

**Issue by Albejr** [bergben/ng2-img-max#30]
### Summary

I'm submitting a: 
- [X] bug report
- [ ] feature request
- [ ] question / support request
- [ ] other

### Description
Necessary to compatibility of import paths to new structure.
![image](https://user-images.githubusercontent.com/24717256/39650254-d2054fe6-4fbd-11e8-90b4-77ff43aa05c8.png)

### Demo
Lib versions:
```
Angular CLI: 6.0.0
Node: 8.11.1
OS: win32 x64
Angular: 6.0.0
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, material, platform-browser
... platform-browser-dynamic, platform-server, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.6.0
@angular-devkit/build-angular     0.6.0
@angular-devkit/build-optimizer   0.6.0
@angular-devkit/core              0.6.0
@angular-devkit/schematics        0.6.0
@angular/cdk                      5.2.5
@angular/http                     5.2.10
@ngtools/webpack                  6.0.0
@schematics/angular               0.6.0
@schematics/update                0.6.0
rxjs                              6.1.0
typescript                        2.7.2
webpack                           4.6.0
```

**Comment by thomasaelbrecht** [comment]
When are you planning to release this update?

**Comment by jdmaldonado** [comment]
I just make my first PR (ever)  to a library.
https://github.com/bergben/ng2-img-max/pull/31
With Angular 6 version and trying to fix the Observable issues, 
If someone wants to give me a feedback, It will be appreciated.

I hope it works :)

**Comment by trymtodalshaug** [comment]
Anyone know if there is any progress on the Angular 6 support update?

**Comment by thomasaelbrecht** [comment]
I think @jdmaldonado is working on the update on this. There is an open pull request for this: #31 but I don't know anything about the progress. I hope it will be merged soon.

**Comment by bergben** [comment]
Hey guys! 
Thanks so much for the pull requests @jdmaldonado and @Flowwwww. Sorry for the late merge. 
I just released version 2.1.16 which updates to Angular 6.0.4 and rxjs 6.2.0.

I am looking for a new maintainer for this project (and my other projects, closely connected to this is ng2-img-tools e.g.), if any of you would be interested that'd be great! Let me know!
